NHSN/src/test/java/com/nhsn/testcases/TC06_VerifySubmitDateOnReviewPage.java

```diff
@@ -0,0 +1,42 @@
+package com.nhsn.testcases;
+
+import com.nhsn.BaseClass;
+import com.nhsn.helpers.CommonFunctions;
+import com.nhsn.pages.HomePage;
+import com.nhsn.pages.LoginPage;
+import com.nhsn.pages.ReviewPage;
+import org.openqa.selenium.JavascriptExecutor;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class TC06_VerifySubmitDateOnReviewPage  extends BaseClass {
+
+    CommonFunctions cFunctions;
+    HomePage hPage;
+    ReviewPage rPage;
+
+    @Test
+    public void verifySubmitDate()  {
+        cFunctions = new CommonFunctions(driver);
+        hPage = new HomePage(driver);
+        rPage = new ReviewPage(driver);
+        try
+        {
+            String note = cFunctions.submitTheReport("praveen_manuel", "Welcome@123", "NHSN Medication Administration");
```

Comment:
Credentials should be put in configuration. Should never see plain-text credentials in this git repository.

Reply:
I moved the credentials in to the Environment variables like how we did for Browser stack